src/recon_buildblock/CUDA/CudaGibbsPenaltiesInstantiations.cu

```diff
@@ -0,0 +1,17 @@
+// Minimal CUDA TU to provide explicit template instantiations for Gibbs priors
+
+/*
+   SPDX-License-Identifier: Apache-2.0
+*/
+
+#include "stir/recon_buildblock/GibbsQuadraticPenalty.h"
+#include "stir/recon_buildblock/GibbsRelativeDifferencePenalty.h"
+#include "stir/recon_buildblock/CUDA/CudaGibbsPenalty.h"
+
+START_NAMESPACE_STIR
+
+// Ensure the CUDA base template is instantiated for the potentials we use
+template class CudaGibbsPenalty<float, QuadraticPotential<float>>;
+template class CudaGibbsPenalty<float, RelativeDifferencePotential<float>>;
```

Comment:
I don't think we need this really. The compiler will instantiate them because we use them. These days I'm in favour of avoiding it, as it creates extra code to maintain.